app/assets/javascripts/govuk_publishing_components/components/autocomplete.js

```diff
@@ -0,0 +1,561 @@
+//= require ../vendor/polyfills/closest.js
+/* global fetch */
+
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {}
+window.GOVUK.Modules.GovukAutocomplete = window.GOVUKFrontend.Autocomplete;
+
+(function (Modules) {
+  'use strict'
+
+  var util = {
+    getRelativePosition: function (element1, element2) {
```

Comment:
This function looks like it might be used to determine if the autocomplete result list should be appended above or below the text input, is that right? I'm not seeing that behaviour in the live example, is this doing something else?